core/domain/user_services.py

```diff
@@ -2038,6 +2038,18 @@ def remove_translation_review_rights_in_language(user_id, language_code):
     _update_user_contribution_rights(user_contribution_rights)
 
 
+def remove_blog_editor(user_id):
+    """Removes the role of user as blog editor.
+
+    Args:
+        user_id: str. The unique ID of the user.
+    """
+    user_settings = get_user_settings(user_id, strict=True)
+    if feconf.ROLE_ID_BLOG_POST_EDITOR == user_settings.role:
```

Comment:
@DubeySandeep aren't we allowing multiple roles for a user? If that PR isn't in yet, note that you will need to change this later.

Reply:
Ack, in my PR I've replaced user_settings.role with roles so this will fail and I'll fix it!